x-pack/package.json

```diff
@@ -54,7 +53,7 @@
     "@types/mocha": "^5.2.5",
     "@types/pngjs": "^3.3.1",
     "@types/prop-types": "^15.5.3",
-    "@types/react": "16.3.14",
+    "@types/react": "^16.3.14",
```

Comment:
We'll be dependent on newer types (&etc) such as `FunctionComponent` which `SFC` is deprecated in favor of. These will require a minimum version of (at least) `16.7`, these dependency version strings should reflect that minimum.

Reply:
good catch, thanks

Reply:
I have narrowed the version to `^16.8.0`, because this will represent the actual minimum version as soon as the first hook usage will be committed.